userguide/dailyuse/anbox.rst

```diff
@@ -97,19 +105,19 @@ Keep your apps up to date
 How to uninstall apps
 ---------------------
 
-- To uninstall apps, run ``adb uninstall [APP_ID]`` from your Ubuntu Touch device::
+- To uninstall apps, run ``adb uninstall [APP_ID]`` from your computer::
 
-    adb shell # connect from your host computer to your UT device
+    adb shell # connect from your computer to your Ubuntu Touch device
     sudo mount -o rw,remount /
-    adb uninstall [APP_ID] # This is the adb of your device, not your host
+    adb uninstall [APP_ID] # This is the adb of your device, not your computer
     exit
 
-- Done! You might have to refresh the apps scope (pull down from the center of the screen and release) for the new Android apps to show up.
+- Done! You might have to pull down from the center of the screen and release for the new Android apps to show up.
```

Comment:
I think we should say **where** to do that. If you are in the settings app, then "pull down from the center of the screen" isn't gonna have the desired effect. I kinda liked the focus on "refresh", rather than the mechanics of swipe interface. How about

You might have to refresh the list of apps for the new Android apps to show up. (Pull down from the center of the apps scope and release.)

Reply:
@doniks Does it refresh by itself or not? "Apps scope" is not good. Give me anything else to describe it by.

Reply:
app overview, app icons, app icons screen, app list

Reply:
or remove the mechanics completely and gamble that users know/will figure it out :)

Reply:
```suggestion
- Done! You might have to pull down from the app list for the new Android apps to show up.
```